docs/reference/release-notes/v2.28/index.md

```diff
@@ -172,4 +172,4 @@ module.exports = {
 
 ## Contributors
 
-A big **Thank You** to [everyone who contributed](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.28.0-next.0...gatsby@2.28.0) to this release 💜
+A big **Thank You** to [the 30 community members who contributed](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.28.0-next.0...gatsby@2.28.0) to this release 💜
```

Comment:
Is this correct number? That's list I got (github usernames):
```
ascorbic
blainekasten
gillkyle
hassankhan
jamo
jHards
jmiazga
kimbaudi
KKVANONYMOUS
KyleAMathews
laurieontech
leerob
LekoArts
mfrachet
mxstbr
nina-py
oorestisime
pieh
pvdz
rburgst
styxlab
theskillwithin
visualfanatic
vladar
wardpeet
```
And that's 26 after removing bots (gatsbybot, renovate)

Reply:
This is where I pulled it from 🤷 I assumed that was accurate but maybe not. Perhaps the command doesn't show people w/o github usernames or I think people can choose to be anonymous?

<img width="1540" alt="Screen Shot 2020-12-02 at 2 48 23 PM" src="https://user-images.githubusercontent.com/71047/100940853-8f1d9800-34ad-11eb-951c-385853b42c8c.png">